TheHyruleCastAdventureGame.js

```diff
@@ -0,0 +1,536 @@
+const readlineSync = require("readline-sync");
+const chalk = require("chalk");
+
+console.log(
+  chalk.redBright.italic(
+    `You awaken in the middle of what seems\nto look like Central Hyrule.`
+  )
+);
+console.log(
+  chalk.redBright.italic(
+    `Far away, you can see The Hyrule Castle standing in front of you with a weird red energy surrounding it.`
+  )
+);
+console.log(
+  chalk.redBright.italic(`it sounds like Princess Zelda but you can't see her.`)
+);
+let nameOfPlayer = readlineSync.question(
+  chalk.magenta.bold(`What is your name hero ?`)
+);
+console.log(
+  chalk.redBright.italic(
+    `${nameOfPlayer}!  It's me Zelda, Ganon has me captive in The Study Hall in my quarters.\nPlease help me ${nameOfPlayer}!!!!`
+  )
+);
+console.log(
+  chalk.redBright.italic(
+    `the voice fades away with the words "Please Help me" echoing over and over`
+  )
+);
+console.log(
+  chalk.redBright.italic(`Welcome to The Hyrule Castle Adventure Game`)
+);
+console.log(
+  chalk.redBright.italic(
+    `In this game ${nameOfPlayer}, you will have to go through the castle and try to save princess Zelda.`
+  )
+);
+console.log(
+  chalk.redBright.italic(
+    `Your choices in this game are very important as you can either save the princess easily or you might not...`
+  )
+);
+console.log(
+  chalk.redBright.italic(
+    `Be careful of deadends and ways that sends you flying out of the castle.`
+  )
+);
```

Comment:
Try thinking of a way to create a for-loop that will display all of these messages. The for-loop block will contain a  single console.log and we will pass a array element that holds a string message.

Reply:
re-did this one, but did it in just an easy way to read instead of a for loop